### ISSUE TITLE
Names, Portraits and Sounds not Swapping

### ISSUE THREAD

**Issue by AstroIsAlive** [lashoun/upgraded-fefates-randomizer#11]
The mod itself seems to work but I've noticed the portraits and sounds are not swapping to the character they've swapped with. I don't see anything wrong with my process so i wanted to know if that's not in this version of the randomizer or if I'm missing something.

**Comment by lashoun** [comment]
Did you tick all sub-boxes below "Join Order" in the .jar executable? "Sounds" and "Models" in particular.

**Comment by AstroIsAlive** [comment]
> Did you tick all sub-boxes below "Join Order" in the .jar executable? "Sounds" and "Models" in particular.

Yes, I ticked everything but the same sex box as I wanted anyone to be swapped around

**Comment by lashoun** [comment]
Sorry, I don't have enough information to help you more. Closing as stale.
